packages/webapp/src/containers/Finances/ActualRevenue/index.jsx

```diff
@@ -32,13 +36,18 @@ export default function ActualRevenue({ history, match }) {
     mode: 'onBlur',
     shouldUnregister: true,
     defaultValues: {
-      from_date: dateRange.startDate,
-      to_date: dateRange.endDate,
+      from_date: dateRange?.startDate
+        ? new Date(dateRange.startDate.split('T')[0] + 'T00:00:00.000Z').toISOString().split('T')[0]
```

Comment:
Not sure exactly what your trying to do, but are you trying to prevent the date being changed by the timezone?

Something like this has worked for me in the past: `new Date(startDate.getUTCFullYear(), startDate.getUTCMonth(), startDate.getUTCDate())`